Client/src/components/layout/Form/FormElement.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+function FormElement(props) {
+  return (
+    <div className={'input-element'}>
```

Comment:
1) You can omit curly braces here for consistency
2) I can see you are using css module approach in other files, css strings here - which one do you stick to?

Reply:
Done. Going with css modules. Resolved, although I'll come back to redo all inputs I have as propper components.